test/test_parse.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+
+import psqlparse
+
+
+class TestParse(unittest.TestCase):
+
+    def test_is_select_statement(self):
+        query = "SELECT * FROM my_table"
+        select_stmt = psqlparse.parse(query).pop()
+        self.assertEqual(select_stmt.type, "SelectStmt")
```

Comment:
I would like a few more checks. Like `with_clause` not set, `target_list` set and so on. Same for next test.